src/assets/src/components/meetingType.tsx

```diff
@@ -2,6 +2,13 @@ import * as React from "react";
 import { Form } from "react-bootstrap";
 
 
+export enum MeetingType {
+    inperson = "In Person",
+    bluejeans = "BlueJeans",
+    zoom = "Zoom"
+}
+
```

Comment:
I'm honestly not sure if this is necessary, but for `VideoMeetingInfo`, it seemed nice not to have to use string comparison to check the backend type, while also being able to use the friendly name stored as a value.